test-suite/squidconf/subnetwork.conf

```diff
@@ -0,0 +1,8 @@
+## Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+acl foo src 192.168.0.0/16 192.168.1.0/24
```

Comment:
Let's also add one tricky positive test where _no_ problems should be reported. That test may contain a combination of IPv4 addresses, IPv6 addresses, domain names, subdomains, and IP address ranges. That are as close to being duplicated (in all forms) as possible while still staying positive.

Reply:
I added several such cases into acl-src-dstdomain-http-status.conf for now. We can add more, if it is not 'tricky' enough.